### DISCUSSION TITLE
Multiple download clients

### DISCUSSION THREAD

**Discussion by Drazzilb08** [cross-seed/cross-seed#147] [Q&A] [answered]
So I decided to split up my downloads through multiple download clients (due to having so many torrents). Now I'm looking for a solid way to have cross seed work across multiple clients.

So I have a qbit for radarr and one for sonarr. Now the idea is to have cross-seed search the first instance then move onto the second one. 

I'm running the containerized version on Unraid.

**Comment by mmgoodnow** [comment, marked as answer]
Support for multiple clients is possible on an infinite timescale. Until then I recommend using multiple instances of cross-seed.

**Reply by Drazzilb08** [reply]
Thank you kindly sir.
